app/controllers/nonsipp/loansmadeoroutstanding/LoansListController.scala

```diff
@@ -131,7 +130,7 @@ class LoansListController @Inject()(
   def onSubmit(srn: Srn, page: Int, mode: Mode): Action[AnyContent] = identifyAndRequireData(srn) { implicit request =>
     loanRecipients(srn).map { recipients =>
       if (recipients.length == maxLoans) {
-        Redirect(navigator.nextPage(AccountingPeriodListPage(srn, addPeriod = false, mode), mode, request.userAnswers))
+        Redirect(navigator.nextPage(LoansListPage(srn, addLoan = false), mode, request.userAnswers))
```

Comment:
Misc bugfix, unrelated to statuses - using page from Loans section instead of Accounting Periods.